src/parser.rs

```diff
@@ -1593,6 +1593,30 @@ impl Parser {
         })
     }
 
+    pub fn parse_set(&mut self) -> Result<Statement, ParserError> {
```

Comment:
Oh, this will be fun. I've been holding off adding support for [MSSQL interpretation of `SET`](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-statements-transact-sql?view=sql-server-2017).

Reply:
Oh boy. We may need to start thinking more about dialect-specific code.